go.mod

```diff
@@ -2,4 +2,7 @@ module go.coder.com/cli
 
 go 1.12
 
-require github.com/spf13/pflag v1.0.3
+require (
+	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.6.1
```

Comment:
We use https://pkg.go.dev/cdr.dev/slog@v1.3.0/sloggers/slogtest/assert?tab=doc instead of testify in the enterprise repo. I think it would make sense to use it here too!